app/scripts/views/sign_up.js

```diff
@@ -197,7 +198,23 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
     },
 
     suggestEmail: function () {
-      mailcheck(this.$el.find('.email'), this.metrics, this.translator, this.window.location.search);
+      var abChoose;
+      var abData = {
+        isMetrics: !! this.metrics._able,
```

Comment:
Maybe we should add a function to this.metrics - `isNull` that indicates "metrics are not enabled"